src/style/style_image.js

```diff
@@ -52,9 +52,9 @@ export function renderStyleImage(image: StyleImage) {
  * can be used to update the image.
  *
  * @interface StyleImageInterface
- * @property {number} width
- * @property {number} height
- * @property {Uint8Array | Uint8ClampedArray} data
+ * @property {number} width Width in pixels.
```

Comment:
For readability, I like the [require-hyphen-before-param-description](https://github.com/gajus/eslint-plugin-jsdoc#eslint-plugin-jsdoc-rules-require-hyphen-before-param-description) rule, but it's tangential and not related to the PR under consideration.

```suggestion
 * @property {number} width - Width in pixels.
```

Reply:
That does look nicer! Perhaps we should make a separate PR/issue?